root/layout/header/top.tt

```diff
@@ -2,16 +2,10 @@
     <ul class="menu">
     [%- IF c.user_exists -%]
         <li class="account">
-            [% link_entity(c.user) %]
+            <span class="menu-header">[%- c.user.name -%]</span>
```

Comment:
No code for that AFAICT, so would have to be factored out of `link_editor`. I guess it would be nice but isn't a big deal.